src/rpdk/core/data_loaders.py

```diff
@@ -52,24 +51,27 @@ def copy_resource(package_name, resource_name, out_path):
         shutil.copyfileobj(fsrc, fdst)
 
 
-def make_validator(schema, base_uri=None, timeout=TIMEOUT_IN_SECONDS):
+def make_validator(schema, base_uri=None):
     if not base_uri:
         base_uri = Draft7Validator.ID_OF(schema)
 
-    def get_with_timeout(uri):
-        return requests.get(uri, timeout=timeout).json()
+    def get_from_local(uri):  # pylint: disable=unused-argument
+        meta_schema = Path(os.path.dirname(os.path.realpath(__file__))).joinpath(
+            "data/schema/meta-schema.json"
+        )
+        return json.load(meta_schema.open())
 
     resolver = RefResolver(
         base_uri=base_uri,
         referrer=schema,
-        handlers={"http": get_with_timeout, "https": get_with_timeout},
+        handlers={"http": get_from_local, "https": get_from_local},
```

Comment:
why do you need protocols if it's reading from the disk now?

Reply:
As the reference in the schema is still pointing to the external HTTPS link.  So the handler is trying to override the protocol handler to retrieve from disk.